project/core/templates/static_nav.html

```diff
@@ -13,20 +13,20 @@
 
         <!-- Right Nav -->
         <ul class="right hide-on-med-and-down">
-            <li><a class="dark-white-text menu-text" href="howitworks">{% trans "How CiviWiki Works" %}</a></li>
-            <li><a class="dark-white-text menu-text" href="about">{% trans "About Us" %}</a></li>
-            <li><a class="dark-white-text menu-text" href="support_us">{% trans "Support Us" %}</a></li>
-            <li><a href="login" class="waves-effect waves-light btn login-btn">{% trans "Login / Sign Up" %}</a></li>
+            <li><a class="dark-white-text menu-text" href="{% url 'how it works' %}">{% trans "How CiviWiki Works" %}</a></li>
+            <li><a class="dark-white-text menu-text" href="{% url 'about' %}">{% trans "About Us" %}</a></li>
+            <li><a class="dark-white-text menu-text" href="{% url 'support us' %}">{% trans "Support Us" %}</a></li>
```

Comment:
Let's change these to camel case here and update the related `urls.py` file.
```suggestion
            <li><a class="dark-white-text menu-text" href="{% url 'how_it_works' %}">{% trans "How CiviWiki Works" %}</a></li>
            <li><a class="dark-white-text menu-text" href="{% url 'about' %}">{% trans "About Us" %}</a></li>
            <li><a class="dark-white-text menu-text" href="{% url 'support_us' %}">{% trans "Support Us" %}</a></li>
```

https://github.com/CiviWiki/OpenCiviWiki/blob/3c3282cd091246b041adc6c71099a288d4a77693/project/frontend_views/urls.py#L6-L9